ui.apps/src/main/content/jcr_root/apps/aem-dictionary-translator/content/dictionaries/message-entries/.content.xml

```diff
@@ -36,8 +36,8 @@
                           src.uritemplate="/apps/aem-dictionary-translator/content/granite/dialog/dictionary/key/create.html${requestPathInfo.suffix}"/>
                     <granite:rendercondition
                             jcr:primaryType="nt:unstructured"
-                            sling:resourceType="granite/ui/components/coral/foundation/renderconditions/simple"
-                            expression="${granite:contains(granite:relativeParent(requestPathInfo.suffix, 0), '/content')}"/>
+                            sling:resourceType="aem-dictionary-translator/components/rendercondition/editable-dictionary"
```

Comment:
should this reside under components?

Reply:
is the default for render conditions, is now a servlet